src/index.js

```diff
@@ -406,9 +406,13 @@ class AvatarEditor extends React.Component {
 
   loadImage(image) {
     if (isFileAPISupported && image instanceof File) {
-      loadImageFile(image).then(this.handleImageReady)
+      loadImageFile(image)
+        .then(this.handleImageReady)
+        .catch(errEvent => this.props.onLoadFailure(errEvent))
```

Comment:
you can just go for

`.catch(this.props.onLoadFailure)`

Reply:
Yup, definitely. I'll do it today.

Reply:
'today' has come!

Reply:
thanks!